core/fake/domain_storage.go

```diff
@@ -18,38 +18,38 @@ import (
 	"context"
 	"fmt"
 
-	"github.com/google/keytransparency/core/adminstorage"
+	"github.com/google/keytransparency/core/domainstorage"
 )
 
 // AdminStorage implements adminstorage.Storage
 type AdminStorage struct {
```

Comment:
Ought you to rename `AdminStorage` -> `DomainStorage` here too?

Reply:
Yes. Fixed.